baseband/mark4/base.py

```diff
@@ -513,39 +517,40 @@ def write(self, data, invalid_data=False):
 
 
 open = make_opener('Mark4', globals(), doc="""
---- For reading a stream : (see `~baseband.mark4.base.Mark4StreamReader`)
+--- For reading a stream: (see :class:`~baseband.mark4.base.Mark4StreamReader`)
 
+sample_rate : `~astropy.units.Quantity`, optional
+    Number of complete samples per second (i.e. the rate at which each channel
+    is sampled).  If not given, will be inferred from scanning two frames of
+    the file.
 ntrack : int, optional
-    Number of tracks used to store the data. Will be inferred from file
-    if not given.
+    Number of Mark 4 bitstreams.  If `None`, will attempt to automatically
+    detect it by scanning the file.
 decade : int, or None, optional
     Decade of the observation start time (eg. ``2010`` for 2018), needed to
     remove ambiguity in the Mark 4 time stamp.  Can instead pass an approximate
     `ref_time`.
 ref_time : `~astropy.time.Time`, or None, optional
     Reference time within 4 years of the start time of the observations.  Used
-    only if `decade` is ``None``.
-subset : slice or other indexing object, optional
-    Specific components (ie. channels) of the complete sample to decode.  By
+    only if `decade` is `None`.
+subset : indexing object, optional
+    Specific components (i.e. channels) of the complete sample to decode.  By
     default, all channels are read.
-sample_rate : `~astropy.units.Quantity`, optional
-    Number of complete samples per second (ie. the rate at which each channel
-    is sampled).  If not given, will be inferred from scanning two frames of
-    the file.
 squeeze : bool, optional
     If `True` (default), remove any dimensions of length unity from
     decoded data.
 
---- For writing a stream : (see `~baseband.mark4.base.Mark4StreamWriter`)
+--- For writing a stream: (see :class:`~baseband.mark4.base.Mark4StreamWriter`)
```

Comment:
I think the space was actually needed for much nicer typesetting (as it makes it look like an argument -- please try; obviously, this affects all formats).

Reply:
If the space is left in the line goes to 80 characters, which is why I originally removed it.  I'll remove `:class:` instead (which I never cared for, but does make the docstrings inconsistent).